### PR TITLE
Query History: split file

### PR DESCRIPTION
**Pull request by norascheuch**
1 class 1 file

Changes are quite superficial, only moving both classes around.

## Checklist

- [ ] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [ ] Issues have been created for any UI or other user-facing changes made by this pull request.
- [ ] _[Maintainers only]_ If this pull request makes user-facing changes that require documentation changes, open a corresponding docs pull request in the [github/codeql](https://github.com/github/codeql/tree/main/docs/codeql/codeql-for-visual-studio-code) repo and add the `ready-for-doc-review` label there.

### CODE REVIEW

File: extensions/ql-vscode/src/query-history/query-history-manager.ts [hunk 1]
```diff
@@ -70,9 +65,10 @@ import { QueryRunner } from "../queryRunner";
 import { VariantAnalysisManager } from "../remote-queries/variant-analysis-manager";
 import { VariantAnalysisHistoryItem } from "./variant-analysis-history-item";
 import { getTotalResultCount } from "../remote-queries/shared/variant-analysis";
+import { HistoryTreeDataProvider } from "./history-tree-data-provider";
 
 /**
- * query-history.ts
+ * history-tree-data-provider.ts
```
Comment (shati-patel) [inline extensions/ql-vscode/src/query-history/query-history-manager.ts:71]: Should this be:

```suggestion
 * query-history-manager.ts
```

File: extensions/ql-vscode/src/extension.ts [hunk 2]
```diff
@@ -1637,7 +1637,7 @@ const checkForUpdatesCommand = "codeQL.checkForUpdatesToCLI";
 /**
  * This text provider lets us open readonly files in the editor.
  *
- * TODO: Consolidate this with the 'codeql' text provider in query-history.ts.
+ * TODO: Consolidate this with the 'codeql' text provider in history-tree-data-provider.ts.
```
Comment (shati-patel) [inline extensions/ql-vscode/src/extension.ts:1640]:
```suggestion
 * TODO: Consolidate this with the 'codeql' text provider in query-history-manager.ts.
```

**Review by shati-patel — APPROVED** [review]
LGTM, just a minor name update 🔍

File: extensions/ql-vscode/src/query-history/query-history-manager.ts (hunk 1 above)
Comment (norascheuch) [reply extensions/ql-vscode/src/query-history/query-history-manager.ts:71]: good catch, thank you!

File: extensions/ql-vscode/src/extension.ts (hunk 2 above)
Comment (norascheuch) [reply extensions/ql-vscode/src/extension.ts:1640]: good catch, thank you!

**Pull request merged**